app/serializers/api/admin/payment_method_serializer.rb

```diff
@@ -1,11 +1,29 @@
 class Api::Admin::PaymentMethodSerializer < ActiveModel::Serializer
-  attributes :id, :name, :type, :tag_list, :tags
+  delegate :serializable_hash, to: :method_serializer
 
-  def tag_list
-    object.tag_list.join(",")
+  def method_serializer
+    if object.type == 'Spree::Gateway::StripeConnect'
+      Api::Admin::PaymentMethod::StripeSerializer.new(object)
+    else
+      Api::Admin::PaymentMethod::BaseSerializer.new(object)
+    end
+  end
+end
+
+module Api::Admin::PaymentMethod
```

Comment:
I would move this to its own file for consistency. Future devs might expect finding the base serializer in the `app/serializers/api/admin/` directory like all the others.

Reply:
Resolved